MobileWallet/Screens/Debug/HomeViewControllerDebugExtension.swift

```diff
@@ -93,7 +93,9 @@ extension HomeViewController {
         }))
 
         alert.addAction(UIAlertAction(title: "Generate test data", style: .default, handler: { (_)in
-            self.generateTestData()
+            let storagePath = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!.appendingPathComponent("test_\(UUID().uuidString)").path
```

Comment:
UI shouldn't set its own path, that's a constant in the lib. Also using `test_ ` in the path is reserved for unit tests not to be used when the app is actually running.